core/templates/pages/collection-player-page/collection-footer/collection-footer.component.html

```diff
@@ -3,19 +3,20 @@
     <div class="col-sm-5">
       <ul>
         <li>
-          <img ng-src="<[$ctrl.getStaticImageUrl('/general/apple.svg')]>" class="collection-footer-img" alt="">
+          <img src="{{ getStaticImageUrl('/general/apple.svg') }}" class="collection-footer-img" alt="">
         </li>
       </ul>
     </div>
     <div class="col-sm-7">
       <div class="float-right">
         <ul>
           <li>
+            <!--TODO(orthodoxparadox): Fix translate-->
```

Comment:
Please do not use this here and everywhere. Refer: https://github.com/oppia/oppia/pull/9790#discussion_r450365566